### DISCUSSION TITLE
how to learn DL ML ?

### DISCUSSION THREAD

**Discussion by intrepid-ani** [mrdbourke/pytorch-deep-learning#828] [Q&A]
i have started with this video https://youtu.be/V_xro1bcAuA?si=4CPtGIO6CCuHsl3S for deep learning is this is good, or if you have better option then please suggest

**Comment by jmarintur** [comment]
Hi @intrepid-ani, if you are total beginner, I'd recommend to go for _Deep Learning Specialization by Andrew Ng_, you can find it on Coursera. There's also a free one, if you have some coding experience, at https://course.fast.ai/ This one It's all about hands-on projects, using top-notch models in real-world tasks.

**Reply by intrepid-ani** [reply]
thank you for your suggestion, my last question is I must drop this and continue from there

**Reply by jmarintur** [reply]
Hi @intrepid-ani, there's no need. Embracing diverse learning resources is a great approach.
